src/main/scala/loamstream/loam/intake/aggregator/AggregatorIntakeConfig.scala

```diff
@@ -0,0 +1,32 @@
+package loamstream.loam.intake.aggregator
+
+import java.nio.file.Path
+import loamstream.conf.ConfigParser
+import scala.util.Try
+import com.typesafe.config.Config
+import loamstream.conf.ValueReaders
+
+/**
+ * @author clint
+ * Mar 2, 2020
+ */
+final case class AggregatorIntakeConfig(
+    condaEnvName: String,
```

Comment:
I'd make the conda environment optional (e.g. `Option[String]`), because in the future we may not use it, or use a different virtual environment, something other than Python, etc. No need for code now to handle the unknown, but I think making conda optional would be simple enough initially?

Reply:
That's a good idea.  I'll do that.